lib/model/index.js

```diff
@@ -39,6 +39,17 @@ function Base() {
   });
 
   /**
+   * Single label support, will mapped to multi label array
+   * 
+   * @name Base#label
+   * @type Object
+   */
+  Object.defineProperty(this, 'label', {
+    get: function() { if (this['labels'][0]) { return this['labels'][0]; } else { return null; }}, //Ensure that the getter returns a label or null
```

Comment:
The reason you need to currently return `null` rather than `undefined` is that parts of the model wiring (especially setting `someElement.labelTarget = label`) does not work with the changes you provide.

I'm currently investigating, if there is an easy fix.